amundsen_common/models/table.py

```diff
@@ -151,6 +153,7 @@ class Table:
     cluster: str
     schema: str
     name: str
+    key: Optional[str] = None
```

Comment:
could you share more what the key is for?

Reply:
Sure! This is the table_uri. We store it on the object in the graph for indexing anyhow, so this saves us from having to recompute it everywhere each time we fetch it.

If we fully roll this out and make it non-optional, it also lets us not recompute the key downstream in the frontend as well: https://github.com/amundsen-io/amundsenfrontendlibrary/blob/ee0474fdcce6c8c1b01c9032b5cd2750b1d0d92c/amundsen_application/api/utils/metadata_utils.py#L110

Reply:
lgtm